src/pages/Follower/Follower.jsx

```diff
@@ -1,37 +1,81 @@
-import { useState } from 'react';
-import { FollowerTopSpan, FollowerMain, FollowerMainUl } from './styled';
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import {
+  FollowerTopSpan,
+  FollowerMain,
+  FollowerMainUl as FollowerUserItem,
+} from './styled';
 import FollowerBtn from '../../components/atoms/Button/Button';
 
 import TopBasicNav from '../../components/CommonUI/Nav/TopBasicNav/TopBasicNav';
 import UserItem from '../../components/CommonUI/UserItem/UserItem';
 import TabMenu from '../../components/CommonUI/TabMenu/TabMenu';
+import { accountNameValue } from '../../atoms';
+import getFollowerListApi from '../../api/profile/getfollower';
+import { follow, unfollow } from '../../api/profile/follow';
+import getProfile from '../../api/profile/getmyprofile';
 
 const Follower = () => {
-  const [tit, setTit] = useState(true);
-  const [color, setColor] = useState(true);
+  const myAccountName = useRecoilValue(accountNameValue);
+  const [followerList, setFollowerList] = useState([]);
 
-  const handleFollowingBtn = () => {
-    setTit(!tit);
-    setColor(!color);
+  const getFollowerList = () => {
+    getFollowerListApi(myAccountName)
+      .then((data) => {
+        setFollowerList(data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
   };
 
-  const toggleColorAndBg = color ? '' : 'active';
+  useEffect(() => {
+    if (myAccountName) getFollowerList();
+  }, [myAccountName]);
+
+  const handleFollowBtn = async (targetName, isfollow) => {
+    if (isfollow) {
+      await unfollow(targetName);
+    } else {
+      await follow(targetName);
+    }
```

Comment:
```suggestion
  const handleFollowBtn = (targetName, isfollow) => {
    if (isfollow) {
      unfollow(targetName);
    } else {
      follow(targetName);
    }
```
api 함수 자체가 비동기로 작성되어 있어서, 여기서 굳이 주지 않으셔도 될 것 같습니다. 저도 이런 부분 많아서 수정할게요.. ㅎ ㅎ

Reply:
수정완료 했습니다!